site/src/lib/blocks.ts

```diff
@@ -53,15 +53,17 @@ export const readBlocksFromDisk = (): ExpandedBlockMetadata[] => {
       packagePath: path.split("/").slice(-3, -1).join("/"),
       ...JSON.parse(fs.readFileSync(path, { encoding: "utf8" })),
     }))
-    .map((metadata: ExpandedBlockMetadata) => ({
-      ...metadata,
-      author: metadata.packagePath.split("/")[0].replace(/^@/, ""),
-      icon: getBlockMediaUrl(metadata.icon, metadata.packagePath),
-      image: getBlockMediaUrl(metadata.image, metadata.packagePath),
-      lastUpdated:
-        buildConfig.find(({ workspace }) => workspace === metadata.name)
-          ?.timestamp ?? null,
-    }));
+    .map((metadata: ExpandedBlockMetadata) => {
+      return {
+        ...metadata,
+        author: metadata.packagePath.split("/")[0].replace(/^@/, ""),
+        icon: getBlockMediaUrl(metadata.icon, metadata.packagePath),
+        image: getBlockMediaUrl(metadata.image, metadata.packagePath),
+        lastUpdated:
+          buildConfig.find(({ workspace }) => workspace === metadata.name)
+            ?.timestamp ?? null,
+      };
+    });
```

Comment:
What's this change for?

Reply:
I wanted to test the output of the block metadata generator. This helps in adding a console.log whenever needed and getting the data for each block.

Reply:
Reverting this change as I don't feel that strongly about having it as a part of this PR.